app/resources/css-default.template

```diff
@@ -194,6 +196,23 @@ input {
   font-size: 13px;
   margin-bottom: .5em;
   max-width: 100%;
+  border: 0;
+  outline: 0;
+  border-bottom: 1px solid rgb(169, 169, 169);
+  background: transparent;
+}
+
+input:focus {
+  border: 0;
+  outline: 0;
+  border-bottom: 2px solid #3b78e7;
+}
+
+input:not([type]):disabled {
```

Comment:
Can you add a comment what kind of elements this rule matches? It looks not obvious.